bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/GrafikEyeKeypadHandler.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lutron.internal.handler;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.lutron.internal.KeypadComponent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler responsible for communicating with GRAFIK Eye QS devices in
+ * a RadioRA 2 or HomeWorks QS System.
+ *
+ * Does not communicate with the scene controller, timeclock controller, or wireless
+ * and EcoSystem occupancy sensors.
+ *
+ * @author Bob Adair - Initial contribution
+ */
+public class GrafikEyeKeypadHandler extends BaseKeypadHandler {
+
+    private static enum Component implements KeypadComponent {
```

Comment:
Can you move that to a another file?

Reply:
I could, but it will only ever be used by GrafikEyeKeypadHandler. Every keypad handler (or CCI handler) defines a different Component enum which encodes the component numbering for that particular series of keypads.  Since there are 8 of them, with another 4-5 still to be created, it seemed cleaner to define them all inside the classes they are used by.